README.md

```diff
@@ -8,20 +8,146 @@ Please sign up through [this form](https://forms.gle/1w8umSbTF4JTPwyj9) if you h
 
 **Can't wait to have you officially join the team!**
 
-### Pick Your Article
+## How to post-edit an article
```

Comment:
As a suggestion, I'm leaving a wiki link here to the meaning of postediting, so as to make it easier for those who do not know the concept to look for it: https://en.wikipedia.org/wiki/Postediting
I think it is a good fit either here on line 11 or on line 15